src/modules/licences/controller.js

```diff
@@ -292,10 +292,27 @@ const getLicenceCompanyByDocumentId = async (request, h) => {
   }
 };
 
+const postLicenceName = async (request, h) => {
+  const { documentId } = request.params;
+  const { documentName, rename } = request.payload;
```

Comment:
I think the value of `rename` can be established without needing an extra payload parameter, along the lines of:
```
const { isEmpty } = require('lodash');
// ...
const { data : { document_name : currentName } } = await documentsClient.findOne(documentId);
const rename = !isEmpty(currentName);
```

Reply:
As discussed in the SLACK the logic should not sit with the UI so this has been implemented as requested